docs/src/examples/components/Tooltip/Usage/TooltipCustomTargetExample.shorthand.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { Tooltip, Button, Divider, Text, Grid, Ref } from '@stardust-ui/react'
+
+class TooltipExample extends React.Component {
```

Comment:
Investigated with @layershifter and this example does not work correctly (try it in maximized view and you'll see that);

**the reason**: unfortunately the refs are not triggering a re-render so we need to use state to do that ourselves as clients.. here is the correct example that also needs the change you're mentioning [here](https://github.com/stardust-ui/react/pull/1455#discussion_r292454499), which is correct and needed BTW:

```jsx
import * as React from 'react'
import { Button, Divider, Grid, Ref, Text, Tooltip } from '@stardust-ui/react'

const TooltipExample = () => {
  const [target, setTarget] = React.useState<HTMLElement>(null)

  return (
    <Grid columns="auto 1fr">
      {/* CUSTOM DOM ELEMENT is used as target for Tooltip */}
      <Tooltip
        target={target}
        trigger={<Button icon="question" circular styles={{ cursor: 'pointer' }}/>}
        content="well, yes, I am just a garbish text ¯\_(ツ)_/¯"
        position="below"
      />
      <div style={{ marginLeft: 10 }}>
        <Text>Could you guess what does this text means? :)</Text>
        <Divider/>
        <Ref innerRef={setTarget}>
          <Text>
            "To the lascivious looking-glass I, that love's majesty to strut before a want love's
            majesto, to the souls of York."
          </Text>
        </Ref>
      </div>
    </Grid>
  )
}

export default TooltipExample
```